addons/common/XEH_postInit.sqf

```diff
@@ -540,21 +540,30 @@ GVAR(deviceKeyCurrentIndex) = -1;
 [0xC7, [true, false, false]], false] call CBA_fnc_addKeybind;  //SHIFT + Home Key
 
 
-["ACE3 Weapons", QGVAR(unloadWeapon), LLSTRING(unloadWeapon), {
-    // Conditions:
-    if !([ACE_player, objNull, ["isNotInside"]] call FUNC(canInteractWith)) exitWith {false};
+["ACE3 Weapons", QGVAR(unloadWeapon), LSTRING(unloadWeapon), {
+    private _unit = ACE_player;
 
-    private _currentWeapon = currentWeapon ACE_player;
-    if !(_currentWeapon != primaryWeapon _unit && {_currentWeapon != handgunWeapon _unit} && {_currentWeapon != secondaryWeapon _unit}) exitWith {false};
+    // Conditions
+    if !([_unit, objNull, ["isNotInside"]] call FUNC(canInteractWith)) exitWith {false};
```

Comment:
Does this prevent passengers from unloading their primary/secondary/handgun weapons?

Reply:
> Does this prevent passengers/FFV from unloading their primary/secondary/handgun weapons?

I tested with the RHS BMP-1 and RHS M1232: You can unload in FFV positions, but not when inside as a passenger.
You can't reload your weapon using the reload keybind or action menu, so why should you be able to unload using a keybind? That's my train of thought.

Reply:
Ok, as long as FFV is supported that's fine